### DISCUSSION TITLE
Small PB admin UI improvement

### DISCUSSION THREAD

**Discussion by pawelwos** [pocketbase/pocketbase#1796] [Show and tell]
Hello Everyone!

I'm new to PB and I've just started using it on my side project and I must say I really enjoy it :smile: The admin UI is a great tool but for my particular scenario it was missing a thumbnail preview for relation fields...It works nice with file fields types but when I have a media collection and I do a relation field to that collection... it only shows the filename. But it was very easy and simple tweak to the **RecordInfo.svelte** UI's file that adds the previews:

```svelte
<script>
    import ApiClient from "@/utils/ApiClient";
    import CommonHelper from "@/utils/CommonHelper";
    import tooltip from "@/actions/tooltip";
    import RecordFileThumb from "@/components/records/RecordFileThumb.svelte";

    
    export let record;
    export let displayFields = [];

    let imagePreviewTypes = ['jpg', 'jpeg', 'png', 'gif', 'svg']

    $: displayValue = CommonHelper.displayValue(record, displayFields);
</script>

<div class="record-info">
    <i
        class="link-hint txt-sm ri-information-line"
        use:tooltip={{
            text: CommonHelper.truncate(
                JSON.stringify(CommonHelper.truncateObject(record), null, 2),
                800,
                true
            ),
            class: "code",
            position: "left",
        }}
    />
    {#if imagePreviewTypes.includes(displayValue.split('.').pop())}
        <RecordFileThumb {record} filename={displayValue} size="md" /> 
        {#if record.title || record.name}
            <span class="txt txt-ellipsis">{CommonHelper.truncate(record.title ?? record.name, 150)}</span>
        {/if}
    {:else}
    <span class="txt txt-ellipsis">{CommonHelper.truncate(displayValue, 150)}</span>
    {/if}
</div>

<style lang="scss">
    .record-info {
        display: inline-flex;
        vertical-align: top;
        align-items: center;
        max-width: 100%;
        min-width: 0;
        gap: 5px;
        line-height: normal;
        > * {
            line-height: inherit;
        }
    }
</style>

```

It will produce the same thumbnail as the file type field and if the collection has title, name field it will add it as well:

![Screenshot_2023-02-06_17-17-46](https://user-images.githubusercontent.com/435388/217025727-89308ce8-51c7-46db-8e5f-9f361aac4514.png)

... and even for relations selection:

![Screenshot_2023-02-06_17-20-02](https://user-images.githubusercontent.com/435388/217026113-3aa1c578-4360-4e87-8ab4-92e6d0cce431.png)

It might not be useful for everyone but maybe someone finds it helpful :smile: 

Thanks! And keep up the good work! :+1:

**Comment by ganigeorgiev** [comment]
This was recently requested in https://github.com/pocketbase/pocketbase/discussions/1765.

I may consider implementing something similar like the above in the future but for now remains a low priority.
